VirtoCommerce.Domain/Marketing/Model/Promotions/Rewards/CatalogRecurringItemAmountReward.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+namespace VirtoCommerce.Domain.Marketing.Model
```

Comment:
As I understand you need this special reward type for different presentations in the expression builder. 
Right? I just wondering because this type doesn't have any additional properties

Reply:
@tatarincev Correct - I just need a type to react to in the Dynamic Expressions module and in the storefront.